.gitignore

```diff
@@ -57,11 +58,30 @@ target/
 
 # IDE
 .idea/
+.DS_Store
+
+#App files
+wger/config/
+wger/core/migrations
+wger/email/migrations
+wger/exercises/migrations/0004_auto_20190606_1001.py
+wger/exercises/views/exercises_special.py
+wger/gym/migrations/0007_auto_20190606_1001.py
+wger/manager/migrations/0008_auto_20190606_1001.py
+wger/nutrition/migrations/0002_auto_20190606_1001.py
```

Comment:
I am also wondering why you are including migrations in the .ignore file could you highlight on that.